x/rollup/types/events.go

```diff
@@ -3,9 +3,6 @@ package types
 const (
 	AttributeKeyL1DepositTxType   = "l1_deposit_tx_type"
 	AttributeKeyL2WithdrawalTx    = "l2_withdrawal_tx"
-	AttributeKeyBridgedTokenType  = "bridged_token_type"
```

Comment:
Same here, maybe lots of these can be private?

Reply:
For the functional code I think everything but nonce can be private. We may want them public for test assertions however, I'll take a look at existing modules/module unit tests to see how they're exported

Reply:
After looking into it I'm going to leave these as exported for now for use in the work to add `x/rollup` unit tests

Reply:
Sentinel Errors often want to be exported so that calling code can make informed decisions based on a specified vocabulary of error types.

Random sample from this codebase:

```
if _, err := e.blockStore.HeaderByHash(payload.BlockHash); errors.Is(err, monomerdb.ErrNotFound) {
		return &eth.PayloadStatusV1{
			Status: eth.ExecutionInvalidBlockHash,
		}, engine.InvalidParams.With(errors.New("block not found"))
	}
```